dist/pinny.js

```diff
@@ -106,13 +110,17 @@
 
                 // only run lockup if another pinny isn't
                 // open and locked the viewport up already
-                !this._activePinnies() && this.$pinny.lockup('lock');
+                if (this._activePinnies()) {
+                    this.$pinny.data('lockup')._trigger('locked');
+                } else {
+                    this.$pinny.lockup('lock');
+                }
 
                 EventPolyfill.on(events.resize, function() {
                     plugin._repaint();
                 });
 
-                if (!this._activePinnies() && iOS7orBelow) {
+                if (!this._activePinnies()) {
```

Comment:
@scalvert @donnielrt You'll notice I removed `needsSpacer` from here because we need iOS 8+ to blur on orientation change.

Donnie mentioned that this is something we'll probably want to test to make sure this change is acceptable for all other platforms (i.e. Android)